include/sapi/tss2_tcti.h

```diff
@@ -165,6 +165,23 @@ TSS2_TCTI_POLL_HANDLE *handles, size_t *num_handles);
 
 typedef TSS2_TCTI_CONTEXT_COMMON_V1 TSS2_TCTI_CONTEXT_COMMON_CURRENT;
 
+#define TCTI_INFO_SYMBOL "Tss2_Tcti_Info"
+
+typedef TSS2_RC (*TSS2_TCTI_INIT_FUNC) (
+    TSS2_TCTI_CONTEXT *tctiContext,
```

Comment:
The project's coding standard says that variables names must be in lowercase and multiple names separated with an underscore character. I know this `tctiContext` camelCase name comes from the TSS SAPI and TCTI spec and it's all over the code base.

I wonder though if these exceptions should be mentioned in the coding standard doc, or is there any attempt to clean those in the spec and code base to make it consistent?

Reply:
Coding standard updated to account for this situation: https://github.com/intel/tpm2-tss/pull/714

Reply:
For completeness: the coding standard has been updated to include a section documenting "exceptions" to our stated policy: 45a711e60c19edb6243f90d9e34f97f57521f2ff. Specifically that TCG uses weird naming conventions for function 
names / prototypes and to remain compliant with the specs we must use their naming.